packages/cli/src/project/attributes/ttk.ts

```diff
@@ -27,13 +27,13 @@ export class TeamsToolkitAttribute implements IProjectAttribute {
         targetDir,
         'package.json',
         'scripts.dev:teamsfx',
-        'npx env-cmd --silent -f .env npm run dev'
+        "NODE_OPTIONS='--inspect=9239' npx env-cmd --silent -f .env npm run dev"
```

Comment:
nitpick: do we want to consider taking out `--silent`? The other day when I was having ttk troubles it was difficult for me to pin down what the errors I was getting were because there was no feedback at all. Can be for discussion later and not in this PR.

Reply:
@aacebo any thoughts? I feel like we're tryna make it as close as possible to what exists, but I'm happy to be more opinionated here.

Reply:
Where does this log show up?  in vscode?

Reply:
Terminal, i assume?

Reply:
seems fine to me to remove the silent option

Reply:
Removed --silent